phys2bids/io.py

```diff
@@ -292,27 +214,81 @@ def read_header_and_channels(filename):
     return header, channel_list
 
 
-def load_txt_ext(filename, chtrig=0):
+def extract_header_items(channel_list, header=[]):
     """
-    Populate object phys_input.
+    Process items header and channel_list depending on the format (acqKnowledge and labchart).
 
-    Extract header and deduce from it the format file,
-    afterwards pass the needed information to
-    the corresponding reading function.
+    Parameters
+    ----------
+    channel_list : list of strings
+        The channels of the recording
+    header: list
+        list that contains file header
+
+    Returns
+    -------
+    interval : list of strings
+        maximun sampling frequency or interval value and unit for the recording
+    orig_units : list of strings
+        contains original channels units
+    orig_names : list of strings
+        contains original channels name
+
+    Raises
+    ------
+    ValueError
```

Comment:
Same as before. Idk which one is better (probably Attribute?), but can you correct it?

Reply:
Maybe it's actually two different types of error instead?